_includes/release-progress.html

```diff
@@ -28,11 +28,11 @@ <h3>MoveIt Iron 2.6</h3>
       <span class="time-line-box time-line-box--red"></span>
       <p>
       <strong class="color-red">Official Release</strong>
-      <small>31 May 2023</small>
```

Comment:
the other dates in this timeline need to be adjusted as well (right above this one). Simply shift the dates by a year like this one